### DISCUSSION TITLE
Search Quotes returns only empty results array

### DISCUSSION THREAD

**Discussion by nilsbgt** [lukePeavey/quotable#178] [Report Downtime]
Hi, the Search Quotes request only returns empty results array as if there were no quotes found to the query. Last Friday everything worked fine but currently only empty arrays are returned, no matter what query I use. 
I use your API for an project for university and have to present my app tomorrow. 

Thank you for your attention to this matter and I look forward to hearing from you soon with an update on the status of the API.

**Comment by lukePeavey** [comment]
Yes I see what you mean. I'll take a look at this
